swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/WorkConfigurator.kt

```diff
@@ -119,41 +119,18 @@ open class WorkConfigurator(
   }
 
   private fun mergeExclusions(global: Set<Exclusion>?, local: Set<Exclusion>?): Set<Exclusion> {
-    if ((global == null || global.isEmpty()) && (local == null || local.isEmpty())) {
```

Comment:
By doing this, we actually never included the runtime exclusions below. Exiting too early.